src/main/kotlin/org/jitsi/jibri/service/impl/FileRecordingJibriService.kt

```diff
@@ -118,11 +120,21 @@ class FileRecordingJibriService(private val fileRecordingParams: FileRecordingPa
     private var processMonitorTask: ScheduledFuture<*>? = null
 
     init {
-        sink = FileSink(fileRecordingParams.recordingDirectory, fileRecordingParams.callParams.callUrlInfo.callName)
+        sink = FileSink(
+            fileRecordingParams.recordingDirectory,
+            fileRecordingParams.callParams.callUrlInfo.callName
+        )
         jibriSelenium.addStatusHandler(this::publishStatus)
     }
 
     override fun start(): Boolean {
+        if (!createIfDoesNotExist(fileRecordingParams.recordingDirectory, logger)) {
```

Comment:
Yeah I agree, I passed it in here because I wanted access to the exception since different failures throw different exceptions.  So you'd think I'd just have `createIfDoesNotExist` throw itself, but I didn't want to have to do the try/catch everywhere where I called it.  So I dunno, I'm a bit torn.  I agree it feels a little weird.

Reply:
if i don't pass the logger in, the helper doesn't do anything useful, so i basically need to copy those 7 lines out wherever i call it, which i wanted to avoid.

Reply:
Why not have FileUtils use its own logger?